main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -492,11 +492,7 @@ public static boolean isECConnectorActive() {
     }
 
     public static boolean isLCConnectorActive() {
-        return getBoolean(R.string.pref_connectorLCActive, false);
-    }
-
-    public static void setLCConnectorActive(final boolean value) {
-        putBoolean(R.string.pref_connectorLCActive, value);
+        return getBoolean(R.string.pref_connectorLCActive, true);
     }
 
```

Comment:
interesting. I see the set function greyed out in AS, but how is the connector activated if the user checks the "Activate" box then?

Reply:
If you are using preference screens only to change settings, you would need no setter methods, as those changes are handled by Android internally. Setter methods are needed only if you want to be able to change a setting from elsewhere in the program, may it be a menu entry or some dialog option (as in map quick settings popup).